::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2018 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Tue, 18 Jul 2017 19:17:24 +0200
Subject: [PATCH] vPDF: Use arm-virtual2 POD config in sample

NOTE: This will be obsoleted when arm-virtual2 vPDF+IDF lands upstream
in Pharos git repo.

Until then, just align the public network configuration used by the
virtual POD with the Enea lab configuration specific to arm-virtual2
(i.e. public network on 10.0.9.0/24).

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 mcp/config/labs/local/idf-virtual1.yaml | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/mcp/config/labs/local/idf-virtual1.yaml b/mcp/config/labs/local/idf-virtual1.yaml
index 35dab9b9..5ad45364 100644
--- a/mcp/config/labs/local/idf-virtual1.yaml
+++ b/mcp/config/labs/local/idf-virtual1.yaml
@@ -42,10 +42,10 @@ idf:
     public:
       interface: 3  # when used, will be fourth vnet interface, untagged
       vlan: native
-      network: 10.16.0.0
-      ip-range: 10.16.0.100-10.16.0.254  # Some IPs are in use by lab infra
+      network: 10.0.9.0
+      ip-range: 10.0.9.100-10.0.9.253  # Some IPs are in use by lab infra
       mask: 24
-      gateway: 10.16.0.1
+      gateway: 10.0.9.254
       dns:
         - 8.8.8.8
         - 8.8.4.4
@@ -61,16 +61,16 @@ idf:
         # Ordered-list, index should be in sync with node index in PDF
         - interfaces: &interfaces
             # Ordered-list, index should be in sync with interface index in PDF
-            - 'ens3'
-            - 'ens4'
-            - 'ens5'
-            - 'ens6'
+            - 'enp1s0'
+            - 'enp2s0'
+            - 'enp3s0'
+            - 'enp4s0'
           busaddr: &busaddr
             # Bus-info reported by `ethtool -i ethX`
-            - '0000:00:03.0'
-            - '0000:00:04.0'
-            - '0000:00:05.0'
-            - '0000:00:06.0'
+            - '0000:01:00.0'
+            - '0000:02:00.0'
+            - '0000:03:00.0'
+            - '0000:04:00.0'
         - interfaces: *interfaces
           busaddr: *busaddr
         - interfaces: *interfaces
